js/zb.js

```diff
@@ -1073,10 +1075,44 @@ module.exports = class zb extends Exchange {
         //         "tid":1718869018
         //     }
         //
-        const timestamp = this.safeTimestamp (trade, 'date');
-        let side = this.safeString (trade, 'trade_type');
-        side = (side === 'bid') ? 'buy' : 'sell';
-        const id = this.safeString (trade, 'tid');
+        // Swap
+        //
+        //     {
+        //         "amount": "0.002",
+        //         "createTime": "1645787446034",
+        //         "feeAmount": "-0.05762699",
+        //         "feeCurrency": "USDT",
+        //         "id": "6902932868050395136",
+        //         "maker": false,
```

Comment:
The `maker` value is not a string, let's use safeValue for it, and do a boolean-ternary.